utils/zerovec/Cargo.toml

```diff
@@ -26,7 +26,9 @@ include = [
 all-features = true
 
 [dependencies]
-serde = { version = "1.0", optional = true, default-features = false, features = ["alloc"] }
+# TODO(1708): Rename this back to "serde"
+dep_serde = { version = "1.0", optional = true, default-features = false, features = ["alloc"], package = "serde" }
```

Comment:
issue: let's not do this, instead let's have serde_serialize and serde_deserialize features

Reply:
We can rename them when we perform the Rust upgrade

Reply:
This lets the API stay constant though, all the renames are crate-internal. If we rename a feature we have to update all incoming dependencies

Reply:
Okay, that's fair.